frontend/src/pages/sites/Sites.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from "react";
+import { Link } from "react-router-dom";
+import { Button, Card } from "react-bootstrap";
+
+import { useSites } from "src/graphql";
+import { LoadingIndicator, SiteLink } from "src/components/fragments";
+import { isAdmin, createHref } from "src/utils";
+import { ROUTE_SITE, ROUTE_SITE_ADD } from "src/constants/route";
```

Comment:
Updated imports (due to the change below)
```suggestion
import { isAdmin } from "src/utils";
import { ROUTE_SITE_ADD } from "src/constants/route";
```